lib/ts/recipe/emailverification/emailVerificationClaim.ts

```diff
@@ -18,12 +22,15 @@ export class EmailVerificationClaimClass extends BooleanClaim {
 
         this.validators = {
             ...this.validators,
-            isVerified: (refetchTimeOnFalseInSeconds = 10, maxAgeInSeconds = 300) => ({
+            isVerified: (refetchTimeOnFalseInSeconds, maxAgeInSeconds) => ({
                 id: this.id,
                 refresh: this.refresh,
                 shouldRefresh: (payload, userContext) => {
                     const DateProvider = DateProviderReference.getReferenceOrThrow().dateProvider;
 
+                    maxAgeInSeconds = maxAgeInSeconds ?? getThresholdAwareDefaultValue(10);
```

Comment:
do not use `??` cause if `maxAgeInSeconds` is `0`, it will not use `0`, and instead use `getThresholdAwareDefaultValue(10)`

Reply:
Please make sure to remove `??` in all your new code

Reply:
[Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) only checks for `null` or `undefined`, so we would be fine here.